src/run/mod.rs

```diff
@@ -0,0 +1,48 @@
+use args::RunSubcommands;
+
+use crate::{
+    build::ensure_web_setup,
+    external_cli::{cargo, wasm_bindgen, CommandHelpers},
+    manifest::package_name,
+};
+
+pub use self::args::RunArgs;
+
+mod args;
+mod serve;
+
+pub fn run(args: &RunArgs) -> anyhow::Result<()> {
+    let cargo_args = args.cargo_args_builder();
+
+    if let Some(RunSubcommands::Web(web_args)) = &args.subcommand {
+        ensure_web_setup()?;
+
+        // If targeting the web, run a web server with the WASM build
+        println!("Building for WASM...");
+        cargo::build::command().args(cargo_args).ensure_status()?;
+
+        println!("Bundling for the web...");
+        wasm_bindgen::bundle(&package_name()?, args.profile())?;
+
+        let port = web_args.port;
+        let url = format!("http://127.0.0.1:{port}");
```

Comment:
```suggestion
        let url = format!("http://localhost:{port}");
```

(Nit) This can technically be replaced by `localhost`. I don't have a preference, so is there one that you prefer?

Reply:
Right, I think I put `127.0.0.1` because you need to put the IP when binding the URL of the webserver, localhost didn't work there for some reason. But of course, for the URL link we can use it. Lemme change that